site/features.md

```diff
@@ -0,0 +1,14 @@
+# NumPy Features
+
+A collection of tutorials, how-tos, and explanations pertaining to built-in
+NumPy functionality.
```

Comment:
This is a larger point, but do we really want to have how-tos and explanations here?

Reply:
Good question - I have no idea. I chose this wording because I thought the "saving-and-loading-arrays" notebook was intended to be more of a how-to than a tutorial, but that could just be because I don't understand [Diataxis](https://diataxis.fr/) that well :)

I agree we can have a higher level discussion about it, but to skirt the issue for now I can just say "notebooks" instead of "tutorials, how-tos, and explanations"